### PR TITLE
CBLiveGetFile - automation

### PR DESCRIPTION
**Pull request by GalRabin**
@yaakovi I will add deprecated before merging this PR.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28173
fixes: https://github.com/demisto/etc/issues/28234

## Description
# Bug
**Script**: `CBLiveGetFile`
**Description**: Part of the commands that the Script depends on is deprecated.

## Deprecated dependencies:
- `cb-file-get`
- `cb-command-create`

## New enhancements in v2 script:
 - Argument endpoint - hostname and IP handled the same.
 - Timeout argument - When creating a session with a sensor Carbon Black fail to create a session with a low timeout, I exposed the ability to change the timeout.
 - Retries if the session isn't created successfully in the first trial (3 times).
 - Adding session close when finishing files download.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests - The test is only integration tests.
- [x] Documentation

### CODE REVIEW

File: Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- contextPath: CbLiveResponse.File.Extension
+  description: File extension.
+  type: String
+script: '-'
+subtype: python3
+system: false
+tags:
+- carbon-black
+- endpoint
+timeout: '0'
```
Comment (yaakovi) [inline Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.yml:88]: missing `must` field with a list of all relevant commands

File: Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+''' COMMAND FUNCTION '''
+
+
+def cb_live_get_file_command(**kwargs) -> Tuple[str, dict, dict]:
+    entry_contexts = cb_live_get_file(**kwargs)
+    human_readable = tableToMarkdown(name=f"Files downloaded from endpoint {kwargs.get('endpoint')}",
+                                     t=build_table_dict(entry_contexts))
+
+    return human_readable, {}, {}
```
[the remark this answers is not in the document]
Comment (GalRabin) [reply Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py:221]: I return it with the file, As it's ready (I return the file with entry context, In earlier steps, so the user will not wait if the first file already downloaded.

File: Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    Args:
+        entry_contexts: List of entry contexts from command "cb-get-file-from-endpoint"
+
+    Returns:
+        list: filtered list with modified headers
+    """
+    root_ec = 'CbLiveResponse.Commands(val.CbCommandID==obj.CbCommandID&&val.CbSessionID==obj.CbSessionID)'
+
+    table = []
+    for file_ec in [ec.get(root_ec, {}) for ec in entry_contexts]:
```
[the remark this answers is not in the document]
Comment (GalRabin) [reply Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py:199]: Right. Changed it.

File: Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+def build_table_dict(entry_contexts: List[dict]) -> List[dict]:
+    """ Create table from all retirieved entry context.
+
+    Args:
+        entry_contexts: List of entry contexts from command "cb-get-file-from-endpoint"
+
+    Returns:
+        list: filtered list with modified headers
+    """
+    root_ec = 'CbLiveResponse.Commands(val.CbCommandID==obj.CbCommandID&&val.CbSessionID==obj.CbSessionID)'
```
[the remark this answers is not in the document]
Comment (GalRabin) [reply Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.py:196]: Changed it, there is not dependency now.

**Comment by GalRabin** [comment]
@kirbles19 
I will appreciate your review on:
- [ ] [Readme](https://github.com/demisto/content/blob/ed4e960c2b43106e71478a7942978189fa209d5d/Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/README.md)
- [ ] [Yaml](https://github.com/demisto/content/blob/7f7b4a36422329111fac988393c2e670c7c76545/Packs/Carbon_Black_Enterprise_Response/Scripts/CBLiveGetFile_V2/CBLiveGetFile_V2.yml)

**Comment by kirbles19** [comment]
@GalRabin Done.

**Review by yaakovi — APPROVED** [review]
deprecate the original script:
* add `deprecated: true`
* update description
* search where we are using the original script and replace with the new one - @idovandijk FYI

**Comment by idovandijk** [comment]
Are we deprecating this? Will I be able to use the new one without breaking BC? Affects https://github.com/demisto/etc/issues/28142

**Comment by GalRabin** [comment]
@idovandijk File modified by new command structure:
- [ ] playbook-Get_File_Sample_From_Path_-_Carbon_Black_Enterprise_Response.yml
- [ ] script-CBLiveFetchFiles.yml

**Pull request merged**